### ISSUE TITLE
bractrace(3) not detected on NetBSD

### ISSUE THREAD

**Issue by krytarowski** [msteveb/jimtcl#89]
The `backtrace`(3) function is in the `libexecinfo` library (`-lexecinfo`).

How to make it detectable? I was trying to specify `LIBS=-lexecinfo` but it fails.

**Comment by msteveb** [comment]
Thanks for the feedback. The autosetup documentation does say that LDFLAGS and LIBS are used during testing, but this is not actually the case. I'll look into this.
In the meantime we can use cc-check-function-in-lib for backtrace. Can you try this in auto.def?

```
if {[cc-check-function-in-lib backtrace execinfo]} {
    define-append LDLIBS [get-define lib_backtrace]
}
```

**Comment by krytarowski** [comment]
I've added this at the tail of `auto.def` and it still doesn't work

**Comment by krytarowski** [comment]
Another issue is with `--disable-jim-regexp` it cannot detect the system version of `regex`(3).

**Comment by krytarowski** [comment]
```
Invoked as: ./configure --prefix=/usr/pkg --exec-prefix=/usr/pkg --docdir=/usr/pkg/share/doc/jimtcl --full --shared --disable-jim-regexp
Failed: gcc -O2 -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
conftest__.c:1:25: fatal error: crt_externs.h: No such file or directory
compilation terminated.
============
The failed code was:
#include <crt_externs.h>
int main(void) {

return 0;
}
============
Failed: gcc -O2 -g -ggdb -I/usr/include conftest__.c -o conftest__
/var/tmp//ccTU9QSV.o: In function `main':
/tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/conftest__.c:3: undefined reference to `backtrace'
============
The failed code was:
extern void backtrace(void);
int main(void) {
backtrace();
return 0;
}
============
Failed: gcc -O2 -g -ggdb -I/usr/include conftest__.c -o conftest__
/var/tmp//ccfLxaFU.o: In function `main':
/tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/conftest__.c:3: undefined reference to `sysinfo'
============
The failed code was:
extern void sysinfo(void);
int main(void) {
sysinfo();
return 0;
}
============
Failed: gcc -O2 -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
conftest__.c: In function 'main':
conftest__.c:3:12: error: size of array '_x' is negative
 static int _x[sizeof(off_t) == 4 ? 1 : -1] = { 1 };
            ^
============
The failed code was:
#include <sys/types.h>
int main(void) {
static int _x[sizeof(off_t) == 4 ? 1 : -1] = { 1 };
return 0;
}
============
Failed: gcc -O2 -g -ggdb -I/usr/include conftest__.c -o conftest__
/var/tmp//ccqjbu50.o: In function `main':
/tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/conftest__.c:3: undefined reference to `_NSGetEnviron'
============
The failed code was:
extern void _NSGetEnviron(void);
int main(void) {
_NSGetEnviron();
return 0;
}
============
Failed: gcc -O2 -g -ggdb -I/usr/include -c -D_GNU_SOURCE -D_POSIX_SOURCE conftest__.c -o conftest__.o
conftest__.c: In function 'main':
conftest__.c:3:13: error: 'environ' undeclared (first use in this function)
 char **ep = environ;
             ^
conftest__.c:3:13: note: each undeclared identifier is reported only once for each function it appears in
============
The failed code was:
#include <unistd.h>
int main(void) {
char **ep = environ;
return 0;
}
============
Failed: gcc -O2 -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
conftest__.c: In function 'main':
conftest__.c:4:1: error: too few arguments to function 'mkdir'
 mkdir("/dummy");
 ^
In file included from conftest__.c:2:0:
/usr/include/sys/stat.h:244:5: note: declared here
 int mkdir(const char *, mode_t);
     ^
============
The failed code was:
#include <sys/types.h>
#include <sys/stat.h>
int main(void) {
mkdir("/dummy");
return 0;
}
============
Failed: gcc -O2 -g -ggdb -I/usr/include conftest__.c -o conftest__
conftest__.c:1:13: warning: conflicting types for built-in function 'sin'
 extern void sin(void);
             ^
/var/tmp//cco38CEQ.o: In function `main':
/tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/conftest__.c:3: undefined reference to `sin'
============
The failed code was:
extern void sin(void);
int main(void) {
sin();
return 0;
}
============
Failed: gcc -O2 -g -ggdb -I/usr/include conftest__.c -o conftest__
/var/tmp//cc0W6HvR.o: In function `main':
/tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/conftest__.c:3: undefined reference to `ERR_error_string'
============
The failed code was:
extern void ERR_error_string(void);
int main(void) {
ERR_error_string();
return 0;
}
============
Failed: gcc -O2 -g -ggdb -I/usr/include conftest__.c -o conftest__
/var/tmp//cc4gvGFU.o: In function `main':
/tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/conftest__.c:3: undefined reference to `TLSv1_2_method'
============
The failed code was:
extern void TLSv1_2_method(void);
int main(void) {
TLSv1_2_method();
return 0;
}
============
```

**Comment by krytarowski** [comment]
`sin`(3) is also interesting..

```
$ nm /usr/lib/libm.so|grep ' sin$'                                  
000000000001cbb0 T sin
```

**Comment by krytarowski** [comment]
After adding `-fno-builtin` to `CFLAGS`, I'm getting:

```
============
Failed: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
/var/tmp//ccdLJYaU.o: In function `main':
/tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/conftest__.c:3: undefined reference to `sin'
============
The failed code was:
extern void sin(void);
int main(void) {
sin();
return 0;                                                                                                                                                    
}
```

**Comment by krytarowski** [comment]
It's ignoring `-lm` in `LDFLAGS` and `LIBS` for me.

**Comment by krytarowski** [comment]
```
$ uname -rms
NetBSD 7.99.39 amd64
```

with pkgsrc-current

**Comment by msteveb** [comment]
Can you run configure with --debug and post config.log?

**Comment by krytarowski** [comment]
```
Invoked as: ./configure --prefix=/usr/pkg --exec-prefix=/usr/pkg --docdir=/usr/pkg/share/doc/jimtcl --full --shared --disable-jim-regexp --debug
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <stdlib.h>
int main(void) {

return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
int main(void) {
long long _x;
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c -fno-unwind-tables conftest__.c -o conftest__.o
============
int main(void) {

return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c -fno-asynchronous-unwind-tables conftest__.c -o conftest__.o
============
int main(void) {

return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <sys/time.h>
int main(void) {

return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <sys/types.h>
int main(void) {

return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <sys/types.h>
#include <sys/socket.h>
int main(void) {

return 0;
}
============
From cache (ok=1): gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <sys/types.h>
int main(void) {

return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <sys/types.h>
#include <netinet/in.h>
int main(void) {

return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <arpa/inet.h>
int main(void) {

return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <netdb.h>
int main(void) {

return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <sys/un.h>
int main(void) {

return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <dlfcn.h>
int main(void) {

return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <unistd.h>
int main(void) {

return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <dirent.h>
int main(void) {

return 0;
}
============
Failed: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
conftest__.c:1:25: fatal error: crt_externs.h: No such file or directory
compilation terminated.
============
The failed code was:
#include <crt_externs.h>
int main(void) {

return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void inet_ntop(void);
int main(void) {
inet_ntop();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void socket(void);
int main(void) {
socket();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void ualarm(void);
int main(void) {
ualarm();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void lstat(void);
int main(void) {
lstat();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void fork(void);
int main(void) {
fork();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void vfork(void);
int main(void) {
vfork();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void system(void);
int main(void) {
system();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void select(void);
int main(void) {
select();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void execvpe(void);
int main(void) {
execvpe();
return 0;
}
============
Failed: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
/var/tmp//cciiFLVS.o: In function `main':
/tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/conftest__.c:3: undefined reference to `backtrace'
============
The failed code was:
extern void backtrace(void);
int main(void) {
backtrace();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void geteuid(void);
int main(void) {
geteuid();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void mkstemp(void);
int main(void) {
mkstemp();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void realpath(void);
int main(void) {
realpath();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void strptime(void);
int main(void) {
strptime();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void isatty(void);
int main(void) {
isatty();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void regcomp(void);
int main(void) {
regcomp();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void waitpid(void);
int main(void) {
waitpid();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void sigaction(void);
int main(void) {
sigaction();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void sys_signame(void);
int main(void) {
sys_signame();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void sys_siglist(void);
int main(void) {
sys_siglist();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void isascii(void);
int main(void) {
isascii();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void syslog(void);
int main(void) {
syslog();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void opendir(void);
int main(void) {
opendir();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void readlink(void);
int main(void) {
readlink();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void sleep(void);
int main(void) {
sleep();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void usleep(void);
int main(void) {
usleep();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void pipe(void);
int main(void) {
pipe();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void getaddrinfo(void);
int main(void) {
getaddrinfo();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void utimes(void);
int main(void) {
utimes();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void shutdown(void);
int main(void) {
shutdown();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void socketpair(void);
int main(void) {
socketpair();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void isinf(void);
int main(void) {
isinf();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void isnan(void);
int main(void) {
isnan();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void link(void);
int main(void) {
link();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void symlink(void);
int main(void) {
symlink();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void fsync(void);
int main(void) {
fsync();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void dup(void);
int main(void) {
dup();
return 0;
}
============
Failed: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
/var/tmp//ccGoTKuE.o: In function `main':
/tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/conftest__.c:3: undefined reference to `sysinfo'
============
The failed code was:
extern void sysinfo(void);
int main(void) {
sysinfo();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <fcntl.h>
int main(void) {

return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <fcntl.h>
int main(void) {
struct flock _x;
return 0;
}
============
From cache (ok=1): gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <sys/types.h>
int main(void) {

return 0;
}
============
Failed: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
conftest__.c: In function 'main':
conftest__.c:3:12: error: size of array '_x' is negative
 static int _x[sizeof(off_t) == 4 ? 1 : -1] = { 1 };
            ^
============
The failed code was:
#include <sys/types.h>
int main(void) {
static int _x[sizeof(off_t) == 4 ? 1 : -1] = { 1 };
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <sys/types.h>
int main(void) {
static int _x[sizeof(off_t) == 8 ? 1 : -1] = { 1 };
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void fseeko(void);
int main(void) {
fseeko();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void ftello(void);
int main(void) {
ftello();
return 0;
}
============
Failed: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
/var/tmp//ccEd6UGM.o: In function `main':
/tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/conftest__.c:3: undefined reference to `_NSGetEnviron'
============
The failed code was:
extern void _NSGetEnviron(void);
int main(void) {
_NSGetEnviron();
return 0;
}
============
Failed: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c -D_GNU_SOURCE -D_POSIX_SOURCE conftest__.c -o conftest__.o
conftest__.c: In function 'main':
conftest__.c:3:13: error: 'environ' undeclared (first use in this function)
 char **ep = environ;
             ^
conftest__.c:3:13: note: each undeclared identifier is reported only once for each function it appears in
============
The failed code was:
#include <unistd.h>
int main(void) {
char **ep = environ;
return 0;
}
============
From cache (ok=1): gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <sys/types.h>
int main(void) {

return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <sys/stat.h>
int main(void) {

return 0;
}
============
Failed: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
conftest__.c: In function 'main':
conftest__.c:4:1: error: too few arguments to function 'mkdir'
 mkdir("/dummy");
 ^
In file included from conftest__.c:2:0:
/usr/include/sys/stat.h:244:5: note: declared here
 int mkdir(const char *, mode_t);
     ^
============
The failed code was:
#include <sys/types.h>
#include <sys/stat.h>
int main(void) {
mkdir("/dummy");
return 0;
}
============
Failed: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
/var/tmp//ccKWJZAd.o: In function `main':
/tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/conftest__.c:3: undefined reference to `sin'
============
The failed code was:
extern void sin(void);
int main(void) {
sin();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__ -lm
============
extern void sin(void);
int main(void) {
sin();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <openssl/ssl.h>
int main(void) {

return 0;
}
============
Failed: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
/var/tmp//ccnQgsBN.o: In function `main':
/tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/conftest__.c:3: undefined reference to `ERR_error_string'
============
The failed code was:
extern void ERR_error_string(void);
int main(void) {
ERR_error_string();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__ -lcrypto
============
extern void ERR_error_string(void);
int main(void) {
ERR_error_string();
return 0;
}
============
Failed: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
/var/tmp//ccnaHIok.o: In function `main':
/tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/conftest__.c:3: undefined reference to `TLSv1_2_method'
============
The failed code was:
extern void TLSv1_2_method(void);
int main(void) {
TLSv1_2_method();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__ -lssl
============
extern void TLSv1_2_method(void);
int main(void) {
TLSv1_2_method();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include -c conftest__.c -o conftest__.o
============
#include <termios.h>
int main(void) {

return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void dlopen(void);
int main(void) {
dlopen();
return 0;
}
============
From cache (ok=0): gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__
============
extern void backtrace(void);
int main(void) {
backtrace();
return 0;
}
============
Compiled OK: gcc -O2 -fno-builtin -g -ggdb -I/usr/include conftest__.c -o conftest__ -lexecinfo
============
extern void backtrace(void);
int main(void) {
backtrace();
return 0;
}
============
================ defines ======================
define AR ar
define AUTODEPS /tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/autosetup/autosetup /tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/autosetup/local.tcl /tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/auto.def /tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/autosetup/system.tcl /tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/autosetup/cc.tcl /tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/autosetup/cc-shared.tcl /tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/autosetup/cc-db.tcl /tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/autosetup/cc-lib.tcl /tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/autosetup/pkg-config.tcl
define AUTOREMAKE /tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77/configure --prefix=/usr/pkg --exec-prefix=/usr/pkg --docdir=/usr/pkg/share/doc/jimtcl --full --shared --disable-jim-regexp --debug
define CC gcc
define CCACHE 
define CCOPTS -fno-unwind-tables -fno-asynchronous-unwind-tables
define CC_FOR_BUILD cc
define CFLAGS -O2 -fno-builtin -g -ggdb -I/usr/include
define CONFIGURE_OPTS --prefix=/usr/pkg --exec-prefix=/usr/pkg --docdir=/usr/pkg/share/doc/jimtcl --full --shared --disable-jim-regexp --debug
define CPP gcc -E
define CPPFLAGS -I/usr/include
define CXX c++
define CXXFLAGS -O2 -fno-builtin -g -ggdb -I/usr/include -g -ggdb
define CXXOPTS 
define C_EXT_OBJS jim-aio.o jim-array.o jim-clock.o jim-eventloop.o jim-exec.o jim-file.o jim-history.o jim-interp.o jim-load.o jim-namespace.o jim-pack.o jim-package.o jim-posix.o jim-readdir.o jim-regexp.o jim-signal.o jim-syslog.o jim-tclprefix.o
define C_EXT_SHOBJS 
define EXEEXT 
define EXTRA_OBJS linenoise.o
define HAVE_ARPA_INET_H 1
define HAVE_BACKTRACE 1
define HAVE_CRT_EXTERNS_H 0
define HAVE_DIRENT_H 1
define HAVE_DLFCN_H 1
define HAVE_DLOPEN 1
define HAVE_DUP 1
define HAVE_ERR_ERROR_STRING 1
define HAVE_EXECVPE 1
define HAVE_FCNTL_H 1
define HAVE_FORK 1
define HAVE_FSEEKO 1
define HAVE_FSYNC 1
define HAVE_FTELLO 1
define HAVE_GETADDRINFO 1
define HAVE_GETEUID 1
define HAVE_INET_NTOP 1
define HAVE_ISASCII 1
define HAVE_ISATTY 1
define HAVE_ISINF 1
define HAVE_ISNAN 1
define HAVE_LFS 1
define HAVE_LINK 1
define HAVE_LONG_LONG 1
define HAVE_LSTAT 1
define HAVE_MKSTEMP 1
define HAVE_NETDB_H 1
define HAVE_NETINET_IN_H 1
define HAVE_OPENDIR 1
define HAVE_OPENSSL_SSL_H 1
define HAVE_PIPE 1
define HAVE_PKG_CONFIG 0
define HAVE_READLINK 1
define HAVE_REALPATH 1
define HAVE_REGCOMP 1
define HAVE_SELECT 1
define HAVE_SHUTDOWN 1
define HAVE_SIGACTION 1
define HAVE_SIN 1
define HAVE_SLEEP 1
define HAVE_SOCKET 1
define HAVE_SOCKETPAIR 1
define HAVE_STDLIB_H 1
define HAVE_STRPTIME 1
define HAVE_STRUCT_FLOCK 1
define HAVE_SYMLINK 1
define HAVE_SYSINFO 0
define HAVE_SYSLOG 1
define HAVE_SYSTEM 1
define HAVE_SYS_SIGLIST 1
define HAVE_SYS_SIGNAME 1
define HAVE_SYS_SOCKET_H 1
define HAVE_SYS_STAT_H 1
define HAVE_SYS_TIME_H 1
define HAVE_SYS_TYPES_H 1
define HAVE_SYS_UN_H 1
define HAVE_TERMIOS_H 1
define HAVE_TLSV1_2_METHOD 1
define HAVE_UALARM 1
define HAVE_UNISTD_H 1
define HAVE_USLEEP 1
define HAVE_UTIMES 1
define HAVE_VFORK 1
define HAVE_WAITPID 1
define HAVE__NSGETENVIRON 0
define JIM_DOCS 1
define JIM_INSTALL 1
define JIM_IPV6 1
define JIM_MATH_FUNCTIONS 1
define JIM_RANDOMISE_HASH 0
define JIM_REFERENCES 1
define JIM_REGEXP 1
define JIM_SSL 1
define JIM_UTF8 1
define JIM_VERSION 77
define LDFLAGS -lm -L/usr/lib -Wl,-R/usr/lib -Wl,-R/usr/pkg/lib
define LDLIBS -lm -lcrypto -lssl  -lexecinfo
define LDLIBS_aio 
define LDLIBS_array 
define LDLIBS_binary 
define LDLIBS_clock 
define LDLIBS_eventloop 
define LDLIBS_exec 
define LDLIBS_file 
define LDLIBS_glob 
define LDLIBS_history 
define LDLIBS_interp 
define LDLIBS_load 
define LDLIBS_mk 
define LDLIBS_namespace 
define LDLIBS_nshelper 
define LDLIBS_oo 
define LDLIBS_pack 
define LDLIBS_package 
define LDLIBS_posix 
define LDLIBS_readdir 
define LDLIBS_readline 
define LDLIBS_regexp 
define LDLIBS_rlprompt 
define LDLIBS_sdl 
define LDLIBS_signal 
define LDLIBS_sqlite3 
define LDLIBS_stdlib 
define LDLIBS_syslog 
define LDLIBS_tclcompat 
define LDLIBS_tclprefix 
define LDLIBS_tree 
define LDLIBS_win32 
define LDLIBS_zlib 
define LD_LIBRARY_PATH LD_LIBRARY_PATH
define LIBS -lexecinfo
define LIBSOEXT .so.0.77
define LINKFLAGS 
define PKG_CONFIG 
define RANLIB ranlib
define SHELL /bin/ksh
define SHOBJ_CFLAGS -fpic
define SHOBJ_LDFLAGS -shared
define SHOBJ_LDFLAGS_R -shared
define SH_CFLAGS -fpic
define SH_LDFLAGS -shared
define SH_LINKFLAGS -rdynamic
define SH_SOEXT .so
define SH_SOEXTVER .so.%s
define SH_SOPREFIX -Wl,-soname,
define SIZEOF_OFF_T 8
define STATIC_EXTS aio array clock eventloop exec file history interp load namespace pack package posix readdir regexp signal syslog tclprefix binary glob nshelper oo stdlib tclcompat tree
define STRIP strip
define STRIPLIBFLAGS --strip-unneeded
define TCL_EXTS 
define TCL_EXT_OBJS binary.o glob.o nshelper.o oo.o stdlib.o tclcompat.o tree.o
define TCL_LIBRARY /usr/pkg/lib/jim
define TCL_PLATFORM_OS netbsd
define TCL_PLATFORM_PATH_SEPARATOR :
define TCL_PLATFORM_PLATFORM unix
define USE_LINENOISE 1
define bindir /usr/pkg/bin
define build x86_64-unknown-netbsd7.99.39
define build_alias 
define builddir /tmp/pkgsrc-tmp/lang/jimtcl/work/jimtcl-0.77
define cross 
define datadir /usr/pkg/share
define docdir /usr/pkg/share/doc/jimtcl
define exec_prefix /usr/pkg
define host x86_64-unknown-netbsd7.99.39
define host_alias 
define includedir /usr/pkg/include
define infodir /usr/pkg/share/info
define jim_ext_aio 1
define jim_ext_array 1
define jim_ext_binary 1
define jim_ext_clock 1
define jim_ext_eventloop 1
define jim_ext_exec 1
define jim_ext_file 1
define jim_ext_glob 1
define jim_ext_history 1
define jim_ext_interp 1
define jim_ext_load 1
define jim_ext_namespace 1
define jim_ext_nshelper 1
define jim_ext_oo 1
define jim_ext_pack 1
define jim_ext_package 1
define jim_ext_posix 1
define jim_ext_readdir 1
define jim_ext_regexp 1
define jim_ext_signal 1
define jim_ext_stdlib 1
define jim_ext_syslog 1
define jim_ext_tclcompat 1
define jim_ext_tclprefix 1
define jim_ext_tree 1
define lib_ERR_error_string -lcrypto
define lib_TLSv1_2_method -lssl
define lib_backtrace -lexecinfo
define lib_dlopen 
define lib_inet_ntop 
define lib_sin -lm
define lib_socket 
define libdir /usr/pkg/lib
define libexecdir /usr/pkg/libexec
define localstatedir /usr/pkg/var
define mandir /usr/pkg/share/man
define prefix /usr/pkg
define sbindir /usr/pkg/sbin
define sharedstatedir /usr/pkg/com
define srcdir .
define sysconfdir /usr/pkg/etc
define target x86_64-unknown-netbsd7.99.39
define tclsh /usr/pkg/bin/jimsh
define top_srcdir .
```

**Comment by msteveb** [comment]
So that looks fine. I see:

`define LDLIBS -lm -lcrypto -lssl  -lexecinfo`

and

`define HAVE_REGCOMP 1`

Apart from the know issue of LIBS and LDFLAGS being ignored, what problems do you still see?

**Comment by krytarowski** [comment]
Excuse me, I got message "Using built-in regexp" and it was misleading to me.

**Comment by msteveb** [comment]
OK. I'll close this ticket then.

**Comment by krytarowski** [comment]
Thank you!
